pkg/datapath/loader/cache.go

```diff
@@ -111,17 +116,24 @@ func newObjectCache(dp datapath.Datapath, nodeCfg *datapath.LocalNodeConfigurati
 	oc := &objectCache{
 		Datapath:         dp,
 		workingDirectory: workingDir,
+		newTemplates:     make(chan string),
```

Comment:
This likely needs to be buffered. /cc @tgraf

Reply:
(or alternatively we need some kind of timeout on the send side to figure out when shutdown is occurring)

Reply:
Top patch resolves this concern.